provider/core/src/lib.rs

```diff
@@ -149,15 +149,14 @@ pub mod prelude {
     pub use crate::resource::ResourceKey;
     pub use crate::resource::ResourceOptions;
     pub use crate::resource::ResourcePath;
+    pub use crate::resource_key;
 
     pub use crate::any::AsDataProviderAnyMarkerWrap;
     pub use crate::any::AsDowncastingAnyProvider;
     #[cfg(feature = "serde")]
     pub use crate::serde::AsDeserializingBufferProvider;
 }
 
-pub use prelude::*;
```

Comment:
Thought: this line shows as deleted. But apparently no one is using it because CI appears happy.